packages/@react-spectrum/checkbox/chromatic/Checkbox.chromatic.tsx

```diff
@@ -61,6 +61,10 @@ storiesOf('Checkbox', module)
   .add(
     'no label',
     () => renderNoLabel({'aria-label': 'This checkbox has no visible label'})
+  )
+  .add(
```

Comment:
we can delete this one in favor putting it in checkbox group chromatic right? which looks like we need to create, I don't see that file yet or any checkbox groups in Checkbox.chromatic.tsx

Reply:
Good catch, done